packages/dev/core/test/unit/Loading/babylon.sceneLoader.tests.temp

```diff
@@ -583,23 +583,23 @@ describe("Babylon Scene Loader", function () {
                 expect(scene.materials.length, "scene.materials.length").to.equal(1);
                 const mesh = scene.getMeshByName("Box001");
                 expect(mesh, "Box001").to.exist;
-                expect(mesh.metadata, "Box001 metadata").to.exist;
-                expect(mesh.metadata.gltf, "Box001 metadata.gltf").to.exist;
-                expect(mesh.metadata.gltf.extras, "Box001 metadata.gltf.extras").to.exist;
-                expect(mesh.metadata.gltf.extras.kind, "Box001 extras.kind").to.equal("nice cube");
-                expect(mesh.metadata.gltf.extras.magic, "Box001 extras.magic").to.equal(42);
+                expect(mesh._internalMetadata, "Box001 metadata").to.exist;
+                expect(mesh._internalMetadata.gltf, "Box001 metadata.gltf").to.exist;
+                expect(mesh._internalMetadata.gltf.extras, "Box001 metadata.gltf.extras").to.exist;
+                expect(mesh._internalMetadata.gltf.extras.kind, "Box001 extras.kind").to.equal("nice cube");

```

Comment:
Shouldn't this still go on metadata instead of _internalMetadata? As extras is user defined data, it feels to me that the user would still want to access it as a public field.

Reply:
You're right, these are used defined data! I have reverted my changes in this file.